spec/spec_helper.rb

```diff
@@ -26,6 +26,10 @@
   PERMISSIONS_PROTECTED_FILE = '0600'.freeze
 end
 
+def min_puppet_version(version)
```

Comment:
Super nitpicky, but `is_puppet_version_greater_than` might be more clear? Feel free to ignore if you like the current name better.

Reply:
I don't have a strong opinion, but I probably prefer the shorter variant, so I'll keep it as-is.